sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1048,6 +1055,11 @@ def __init__(
         to be passed when creating a BigQuery table. These are passed when
         triggering a load job for FILE_LOADS, and when creating a new table for
         STREAMING_INSERTS.
+      with_insert_ids: When using the STREAMING_INSERTS method to write data to
```

Comment:
Let's refer to 'https://cloud.google.com/bigquery/streaming-data-into-bigquery#disabling_best_effort_de-duplication' similar to Java.

Reply:
I've renamed the option to match the name in Java SDK